phases/phase-1/challenges/accept-referral.md

```diff
@@ -0,0 +1,28 @@
+# Accept a referral
+:::tip Note  
+This challenge is only available top users who have been referred from another user via the [Referral challenge](refer.md) and who have already created a post via the [Post challenge](create-post.md). 
```

Comment:
This challenge is only available top users who have been referred from another user via the [Referral challenge](refer.md) and who have already created a post via the [Post challenge](create-post.md). 

->

You will only be eligible for reward in this challenge if you met **all** the below conditions:
- You have not yet made your first post, and
- You have been referred by another user via the [Referral challenge](refer.md) of which the PR has been accepted